deploy/README.md

```diff
@@ -0,0 +1,309 @@
+- **Job Runner** The MediaWiki [JobRunner](https://www.mediawiki.org/wiki/Manual:Job_queue#Cron) service which uses the same Wikibase container as above.
+- **[MariaDB](https://hub.docker.com/_/mariadb)** Database service for MediaWiki and Wikibase.
+- **[Elasticsearch](https://hub.docker.com/r/wikibase/elasticsearch)** Search service used by MediaWiki.
+- **[WDQS](https://hub.docker.com/r/wikibase/wdqs)** Wikidata Query Service allowing SPARQL queries.
+- **[WDQS Frontend](https://hub.docker.com/r/wikibase/wdqs-frontend)** Web Frontend to run SPARQL queries.
+- **[wdqs-proxy](https://hub.docker.com/r/wikibase/wdqs-proxy)** A middle layer for WDQS which serves to filter requests and make the service more secure.
+- **[WDQS Updater](https://www.mediawiki.org/wiki/Wikidata_Query_Service/User_Manual#runUpdate.sh)** Keeps the WDQS data in sync with Wikibase.
+- **[Quickstatements](https://hub.docker.com/r/wikibase/quickstatements)** A web based tool to import and manipulate large amounts of data.
+- **[Traefik](https://hub.docker.com/_/traefik)** A reverse proxy handling TLS termination and ACME certificate renewal.
+
+## Quickstart
+
+### Requirements
+
+> 💡 If you want to run a quick test on a machine that has no public IP address (e.g. your local machine), check our [FAQ item](#can-i-host-wikibase-suite-locally) below.
+
+#### Hardware
+
+- Network connection with a public IP address
+- AMD64 architecture
+- 8 GB RAM
+- 4 GB free disk space
+- Docker 22.0, or greater
+- Docker Compose 2.10, or greater
+
+#### Domain Names
+
+You need three DNS records that resolve to your machines IP address.
+
+- Wikibase e.g. "wikibase.mydomain.net"
+- QueryService e.g. "query.mydomain.net"
+- QuickStatements e.g. "quickstatements.mydomain.net"
+
+### Initial Setup
+
+#### Download the WBS Deployment Kit
+
+Checkout the files from Github and move to the subdirectory `deploy`.
+
+```sh
+git clone https://github.com/wmde/wikibase-release-pipeline
+# TODO update to the correct TAG/BRANCH
+git checkout wmde.20
+cd wikibase-release-pipeline/deploy
+```
+
+#### Initial Configuration
+
+Make a copy of the [configuration template](./template.env) in `wikibase-release-pipeline/deploy`
+
+```sh
+cp template.env .env
+```
+
+Set usernames, passwords and domain names in your newly created `.env` file
+according to the instructions in the comments.
+
+#### Starting
+
+Run the following command from within `wikibase-release-pipeline/deploy`
+
+```sh
+docker compose up --wait
+```
+
+The first start can take a couple of minutes. Wait for your shell prompt to return.
+
+🎉 Congratulations, your Wikibase Suite instance should be up and running. Web
+interfaces are available via HTTPS (port 443) on the domain names you
+configured for Wikibase, WDQS Frontend and Quickstatements.
+
+#### Stopping
+
+To stop, use 
+
+```sh
+docker compose down
```

Comment:
I think this should be `docker compose stop` and `down` reserved for the updating process detailed below.